ccsljava_execution/ccsljava_engine/plugins/org.eclipse.gemoc.execution.concurrent.ccsljavaengine.ui/src/main/java/org/eclipse/gemoc/execution/concurrent/ccsljavaengine/ui/launcher/Launcher.java

```diff
@@ -107,8 +108,11 @@ public void run() {
 						"Cannot instanciate solver from language definition", e));
 			}
 
-			_executionEngine = new ConcurrentExecutionEngine(concurrentexecutionContext, _solver);
-
+			if (runConfiguration.getIsExhaustiveSimulation()) {
+				_executionEngine = new ExhaustiveConcurrentExecutionEngine(concurrentexecutionContext, _solver);
+			}else {
+				_executionEngine = new ConcurrentExecutionEngine(concurrentexecutionContext, _solver);
+			}
```

Comment:
So far we always each one of our engine has its own launcher. Shouldn't we have a dedicated launcher for this new one, which could probably reuse most of the concurrent launcher? But since we don't have any good reuse strategy for such things, I am fine with what you propose here.